include/evmc/evmc.h

```diff
@@ -795,13 +795,14 @@ enum evmc_revision
  *
  * This function MAY be invoked multiple times for a single VM instance.
  *
- * @param instance   The VM instance.
- * @param context    The pointer to the Client execution context to be passed
- *                   to the callback functions. See ::evmc_context.
- * @param rev        Requested EVM specification revision.
- * @param msg        Call parameters. See ::evmc_message.
- * @param code       Reference to the code to be executed.
- * @param code_size  The length of the code.
+ * @param instance   The VM instance. This argument MUST NOT be NULL.
+ * @param context    The pointer to the Host execution context to be passed
```

Comment:
The tests for precompiles go against this. See #283.

Reply:
I forgot I already added some of this shit.